src/PatientChecking/PatientChecking/Views/ViewModels/DashboardViewModel.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace PatientChecking.Views.ViewModels
+{
+    public class DashboardViewModel
+    {
+        public int NumOfPatients { get; set; }
+        public int NumOfPatientsInMonth { get; set; }
+        public int NumOfAppointments { get; set; }
+        public int NumOfAppointmentsInMonth { get; set; }
+        public int NumOfAppointmentsInToday { get; set; }
+
```

Comment:
Remove this empty line

Reply:
I created separate models for patients and appointments to combine all values needed for each, so i removed DashboardViewModel